docs/pages/reference/data-model/measures.mdx

```diff
@@ -257,6 +257,12 @@ with a defined date range.
 
 </WarningBox>
 
+<WarningBox>
+
+Please note that Redshift does not support intervals of type month or year.
+
+</WarningBox>
+
```

Comment:
@morgan-at-cube Do we know how it breaks? E.g., what is the exact error message?

I think we can probably do the following:
* Add the error message and the note on the Redshift data source page.
* Have a more generic note on this page, e.g., "There are known limitations for `rolling_window` support when using Redhsift (link to the data source page)". It's gonna be more extendable this way.